src/core/lua/luabfb.h

```diff
@@ -62,7 +62,7 @@ class CLuaBFB : public CGenFunctionBlock<CBasicFB> {
     bool initialize() override;
 
     bool createInterfaceSpec(const char *paConfigString, SFBInterfaceSpec &paInterfaceSpec) override {
-      paInterfaceSpec = *mInterfaceSpec;
+      paInterfaceSpec = mInterfaceSpec;
```

Comment:
this should be:
`paInterfaceSpec = *paTypeEntry->getInterfaceSpec()`

Reply:
uhm, there's no paInterfaceSpec parameter in createInterfaceSpec, maybe you refer to the constructor ? (I did that in the latest commit)

Reply:
Ok, done